RELEASING.md

```diff
@@ -2,37 +2,48 @@
 
 ## Production Releases
 
----
-
-***Before you begin:*** Please make sure you are set up with 
-[`bundle exec pod trunk`](https://guides.cocoapods.org/making/getting-setup-with-trunk.html) and your CocoaPods
-account is a contributor for all pods. If you need to be added as a
-contributor, please [open a ticket requesting access](https://github.com/square/workflow-swift/issues/new).
-
-For Squares, membership is managed through the `Workflow Swift Owners` registry group. Please request access to that group through Registry. Once you have access, you can register a session with `bundle exec pod trunk` using the group e-mail alias `workflow-swift-owners@squareup.com`.
-
-`bundle exec pod trunk register workflow-swift-owners@squareup.com 'Workflow Swift Owners' --description='Your computer description'`
-
----
-
-> ⚠️ [Optional] To avoid possible headaches when publishing podspecs, validation can be performed before updating the Workflow version number(s). To do this, run the following in the root directory of this repo:
-> ```bash
-> bundle exec pod lib lint Workflow.podspec ViewEnvironment.podspec ViewEnvironmentUI.podspec WorkflowTesting.podspec WorkflowReactiveSwift.podspec WorkflowUI.podspec WorkflowRxSwift.podspec WorkflowReactiveSwiftTesting.podspec WorkflowRxSwiftTesting.podspec WorkflowSwiftUIExperimental.podspec WorkflowCombine.podspec WorkflowCombineTesting.podspec WorkflowConcurrency.podspec WorkflowConcurrencyTesting.podspec
-> ```
-> You may need to `--include-podspecs` for pods that have changed and are depended on by other of the pods.
-
+### Updating the release
 1. Update `WORKFLOW_VERSION` in `version.rb` based on [`semver`](https://semver.org/).
 
 1. Create a PR with the version change and merge to `main`.
 
 1. Create the release on GitHub:
    1. Go to the [Releases](https://github.com/square/workflow-swift/releases) and `Draft a new release`.
-   1. `Choose a tag` and create a tag for the new version. ex: `v1.0.0`
-   1. `Generate release notes`
+   1. `Choose a tag` and create a tag for the new version. ex: `v1.0.0`.
+   1. `Generate release notes`.
    1. Ensure the `Title` corresponds to the version we're publishing and the generated `Release Notes` are accurate.
    1. Hit `Publish release`.
 
-1. Publish to CocoaPods:
+At this point, the new release is available and can be consumed using the Swift Package Manager.
+
+[Square specific] To make the new version available internally, [bump the version](https://go/spm-bump-dependency) through SPM.
```

Comment:
Should we put anything Square specific in a public .md file?

Reply:
I think go links that has the actual details is fine, [this line](https://github.com/square/workflow-swift/pull/309/files#diff-87d6a57a759f2386bf9caf5309b9aa0b9ff0ca70f5e38696e114a1ccdb0e785dR64) about the same thing with CocoaPods was already there.